src/App.tsx

```diff
@@ -1,15 +1,63 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import cn from 'classnames';
+import { Person } from './types/Person';
+import debounce from 'lodash.debounce';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const [hoveredItem, setHoveredItem] = useState<string | null>(null);
+
+  const [listIsVisible, setListIsVisible] = useState(false);
```

Comment:
```suggestion
  const [IsListVisible, setIsListVisible] = useState(false);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig